test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -25,8 +25,8 @@ public void execute() throws IllegalValueException {
         //same word, same case: matched
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
-        //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        //same word, different case: matched
+        assertFindCommandBehavior(new String[]{"aMy"}, Arrays.asList(td.amy));
```

Comment:
You could use Collections.singletonList(td.amy) instead of Arrays.asList(td.amy) to save memory if there is only 1 argument and since you are not using set() method.